solidity/migrations/2_deploy_contracts.js

```diff
@@ -31,7 +33,7 @@ module.exports = async function(deployer, network) {
 
   // Set the stake initialization period to 1 block for local development and testnet.
   if (network === 'local' || network === 'ropsten' || network === 'keep_dev') {
-    initializationPeriod = 1;
+    initializationPeriod = 120;
```

Comment:
do we have to change it here to 120?

Reply:
We should avoid doing that, it will make the local development hard and a bunch of our system tests to fail.